### PR TITLE
treewide: add only one device when appending to TARGET_DEVICES

### PR DESCRIPTION
**Pull request by luaraneda**
I've encountered conflicts several times when rebasing a developing branch (ipq806x) against master.

This will avoid some conflicts when doing a git rebase or merge,
specially when adding support to a a new device.

### CODE REVIEW

File: target/linux/brcm47xx/image/Makefile [hunk 1]
```diff
@@ -519,7 +519,16 @@ define Device/usrobotics-usr5461
...
+define Device/legacy-standard
+  $(Device/standard)
+endef
+TARGET_DEVICES += legacy-standard
+
+define Device/legacy-standard-noloader-gz
+  $(Device/standard-noloader-gz)
+endef
+TARGET_DEVICES += legacy-standard-noloader-gz
+
```
Comment (NeoRaider) [inline target/linux/brcm47xx/image/Makefile]: What's this, these devices are renamed? This looks like an unrelated change that should be a separate commit.

Comment (mkresin) [reply target/linux/brcm47xx/image/Makefile]: It isn't really unrelated, it is just really hard to get from the diff why it is done.

At the moment we have the same image referenced twice. One time in `ifeq ($(SUBTARGET),legacy)` and one time in `ifeq ($(SUBTARGET),mips74k)`.

With this change it is more obvious that the same images should be build for both subtargets.

But I agree, that this change should go into an extra commit due to the rename of the images. In fact, I dropped this chunk from my local branch which I compile test at the moment.

Comment (luaraneda) [reply target/linux/brcm47xx/image/Makefile]: Exactly, the name of the generated images remains the same, it's just a cosmetic change to identify better the subtarget, and to have a  device defined for every append to TARGET_DEVICES.
I understand that the change should be on a separate commit. Do I add another commit or should I create another pull request?

Without defining the devices, the change would be:
```diff
-TARGET_DEVICES += standard standard-noloader-gz
+TARGET_DEVICES += standard
+TARGET_DEVICES += standard-noloader-gz
```

**Comment by mkresin** [comment]
I merged your PR excluding the brcm47xx changes. Please create a new PR for the brcm47xx changes if you are still interested.
